library/build.gradle.kts

```diff
@@ -0,0 +1,14 @@
+check { dependsOn("testAll", "prepareArtifacts") }
+
+android {
+    lintOptions {
+        isAbortOnError = false
+    }
+}
+
+dependencies {
+    api(project(":auth"))
+    api(project(":database"))
+    api(project(":firestore"))
+    api(project(":storage"))
```

Comment:
Do we need `common` here?

Reply:
We didn't have it before and Gradle should take care of downloading those kinds of transitive dependencies.